### PR TITLE
use pixel id instead of adv id

### PR DESCRIPTION
**Pull request by lmolas**
We are now using pixel identifier instead of advertiser identifier.

### CODE REVIEW

File: template.tpl [hunk 1]
```diff
@@ -116,27 +101,23 @@ setCookie('_km', cid, { 'max-age': 30 * 24 * 60 * 60, 'secure': true });
...
+
 var sUrl = 'https://t.kmtx.io/s?' +
-    'aid=' + encodeUriComponent(data.aid) +
+    'pid=' + encodeUriComponent(data.pid) +
     '&cid=' + cid +
     '&eid=' + uuid() +
     '&a=' + (isTimer ? 'close' : data.type) +
-    '&v=gtm_1' +
+    '&ed=' + (eventData === undefined ? '' : encodeUriComponent(eventData)) +
+    '&v=gtm_2' +
```
Comment (martin-magakian) [inline template.tpl]: should we use `gtm_3.0` to match the new kmpx?
but then should we call this `kmpx_3.0` in the `kmpx.js`?

File: template.tpl [hunk 2]
```diff
@@ -116,27 +101,23 @@ setCookie('_km', cid, { 'max-age': 30 * 24 * 60 * 60, 'secure': true });
...
-sendPixel(sUrl, data.gtmOnSuccess, data.gtmOnFailure);
-
-if (data.sid && !isTimer) {
-    var tsUrl = 'https://t.kmtx.io/ts?' +
-        'aid=' + encodeUriComponent(data.aid) +
-        '&sid=' + encodeUriComponent(data.sid);
-
-    sendPixel(tsUrl, data.gtmOnSuccess, data.gtmOnFailure);
-}
+sendPixel(sUrl, data.gtmOnSuccess(), data.gtmOnFailure());
```
Comment (martin-magakian) [inline template.tpl]: Nop ! you want to pass the function as argument, not calling the function.

```
sendPixel(sUrl, data.gtmOnSuccess, data.gtmOnFailure);
```

File: template.tpl (hunk 1 above)
Comment (lmolas) [reply template.tpl]: it is designed to be a float in kmpx.js

File: template.tpl (hunk 2 above)
Comment (lmolas) [reply template.tpl]: ha yes my bad

File: template.tpl (hunk 1 above)
Comment (lmolas) [reply template.tpl]: I will put gtm_3

**Pull request merged**